control-plane-operator/controllers/hostedcontrolplane/kas/deployment.go

```diff
@@ -155,7 +155,7 @@ func ReconcileKubeAPIServerDeployment(deployment *appsv1.Deployment,
 						"/usr/bin/tail",
 						"-c+1",
 						"-F",
-						"/var/log/kube-apiserver/audit.log",
+						fmt.Sprintf("%s/%s", volumeMounts.Path(kasContainerMain().Name, kasVolumeWorkLogs().Name), "audit.log"),
```

Comment:
Is this what you were thinking @csrwng ?

Reply:
Yes, thank you. The reason is that if you change the volume path in one place, you don't have to worry about changing it everywhere else it's referenced.